index.html

```diff
@@ -66,13 +66,45 @@ <h1>The OpenTF Manifesto</h1>
         </p>
 
         <p>
-          This is why the building blocks of the modern Internet, such as Linux and Kubernetes, are handed over
-          to impartial bodies (the Linux Foundation and the Cloud Native Computing Foundation, respectively) who
-          can ensure that those building blocks can form solid and predictable underpinnings for our industry.
+          We believe that the essential building blocks of the modern Internet, such as Linux, Kubernetes, 
+          and Terraform need to be truly open source: that is the only way to ensure
+          that we are building our industry on top of solid and predictable underpinnings.
         </p>
 
         <p class="font-bold">
-          Our goal: ensure Terraform is handed over to an open source foundation.
+          Our goal: ensure Terraform remains truly open source—always.
```

Comment:
```suggestion
          Our goal: ensure Terraform remains truly open-source — always.
```

Reply:
The em dash shouldn't have spaces around it. As for "open source," we are pretty inconsistent with hyphenation, switching back and forth... Should probably rethink it globally and figure out how we want to do it.